src/app/clusters/account-login-server/account-login-delegate.h

```diff
@@ -44,6 +44,8 @@ class Delegate
                                    const chip::CharSpan & tempAccountIdentifierString)                                 = 0;
     virtual void GetSetupPin(char * setupPin, size_t setupPinSize, const chip::CharSpan & tempAccountIdentifierString) = 0;
 
+    virtual uint16_t GetClusterRevision(chip::EndpointId endpoint) = 0;
```

Comment:
Why the extra `chip::`?  This is inside `namespace chip {`.  This applies to lots of files in this PR.